src/ui/sign_tx_common.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "types.h"
+
+#define MAX_NUM_STEPS 13
+
+typedef struct global_item_storage_s {
+    // 34 + \0
+    char dst_address[35];
+    char system_fee[30];
```

Comment:
Can you define some macros to define the sizes of the different strings ? 
Then you could use them as well on tempo variables in `int start_sign_tx(void)` such as `token_amount` etc.

Reply:
Done